### DISCUSSION TITLE
Left & Right Sidebar / CoreUI

### DISCUSSION THREAD

**Discussion by aboutdomtime** [Megabit/Blazorise#2181] [Q&A] [answered]
Would it be possible to have two sidebars on the same page? one on the left and one on the right, similar to CoreUI?  
```
<container> 
  <head> ---------------------------------------------------------- </head>
  <sidebar></sidebar>   <contents></contents>  <sidebar></sidebar>
  <footer> ------------------------------------------------------- </footer>
<container>
```
![coreui](https://user-images.githubusercontent.com/9167476/114033862-430ba700-9843-11eb-8f7f-de3533be33d3.PNG)

**Comment by stsrki** [comment]
How do I show the right sidebar? I cannot find it on https://coreui.io/demo/free/3.4.0/#

**Reply by aboutdomtime** [reply]
https://coreui.io/angular/demo/#/dashboard

**Reply by stsrki** [reply]
Seems like they're different components

![image](https://user-images.githubusercontent.com/900302/114041716-0eb8da80-9885-11eb-9ff6-8dfc064aa882.png)

**Reply by aboutdomtime** [reply]
Okay. Makes sense. Any suggestions on how I would set this up using Blazorise? Or would I need to build out my own component?

**Comment by stsrki** [comment, marked as answer]
Well, I guess you can make it with some custom CSS. Maybe `position: fixed;` and `right: 0;` for start. It's best to inspect how CoreUI does it and try to copy it.
